prisma/schema.prisma

```diff
@@ -62,6 +62,19 @@ model Session {
   @@map(name: "session")
 }
 
+// Represents an Invitation to a user
+model Invitation {
```

Comment:
Are we sure about the naming here? I heard some discussion about calling this a ``Request`` or something.

Reply:
Idc, either name is fine, would it really matter that much? 
Luke I think the discussion you heard had more to do with the frontend